.ado/jobs/universal.yml

```diff
@@ -65,6 +65,8 @@
                   clean: false
                   submodules: false
 
+                - template: ../templates/apply-published-version-vars.yml
```

Comment:
For my own understanding, when do we publish the env variables?

Reply:
in Setup of both PR (windows-vs-2019.yml) and CI (continious.yml)